lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -735,6 +735,12 @@ def determine_optimal_points(frame_idx, coordinate, max_iterations=100):
 
         num_slopes.add(num_slope)
 
+        if len(coordinate) <= 4:
+            raise RuntimeError(
+                "You need at least 5 time points to estimate the number of points to include in "
+                "the fit."
+            )
```

Comment:
don't we have this check already somewhere?

Reply:
We do have a similar check elsewhere, but you don't even get to that check if you run diffusion estimation on an empty track. Instead you are greeted by a `TypeError: expected non-empty vector for x` which is not helpful.